platform/ios/scripts/publish.sh

```diff
@@ -9,10 +9,10 @@ function finish { >&2 echo -en "\033[0m"; }
 trap finish EXIT
 
 #
-# iOS release tag format is `vX.Y.Z`; `X.Y.Z` gets passed in
+# Update the PUBLISH_VERSION manually when releasing a new iOS 8 compatible SDK.
 # In the case of symbolicated builds, we also append the `-symbols`.
 #
-PUBLISH_VERSION="$1"
+PUBLISH_VERSION="v4.5.0-cn.1"
```

Comment:
https://github.com/mapbox/mapbox-gl-native/blob/master/platform/ios/scripts/deploy-packages.sh#L92 The `PUBLISH_VERSION ` can get from `VERSION_TAG`, we don't need set its value to a constant.